bmw_cd_exporter.py

```diff
@@ -37,13 +38,24 @@ def collectData(user, password, region, attributes):
     if DEBUG:
         print('DEBUG: Found {} vehicles: {}'.format(len(account.vehicles),','.join([v.name for v in account.vehicles])))
 
+    if (mqttParams):
+        mqttclient = mqtt.Client(app)
+        mqttclient.connect(mqttParams)
+
+
     # set gauges for each vehicle
     for vehicle in account.vehicles:
         for attr in attrs:
             if getattr(vehicle.state,attr['name']):
-                vehicle_name = ''.join(e for e in vehicle.name if e.isalnum())
-                if gauges[attr['name']]:
-                    gauges[attr['name']].labels(vehicle=vehicle_name, vin=vehicle.vin).set(getattr(vehicle.state,attr['name']))
+                if (mqttParams):
+                    mqtttopic= vehicle.name + '/' + attr['name']
```

Comment:
The topic name should follow standard naming convention. In this case it could be `bmw-connecteddrive/{vehicle name}/{attribute}`. It should also be sanitized to include only letters, numbers, and dashes.

Reply:
ok for the connected-drive. I will check for the sanitization. We cannot use a wildcard in the topic name but spaces is not forbidden. Errors coudl mainly come from attributes.json file

Reply:
The main point about the topic name is that it should be prefixed with `bmw-connecteddrive/`

It could also come from the vehicle name.  The `vehicle_name` variable should already be sanitized.